src/probnum/filtsmooth/statespace/continuous/linearsdemodel.py

```diff
@@ -25,15 +25,15 @@ class LinearSDEModel(continuousmodel.ContinuousModel):
 
     Parameters
     ----------
-    driftmatrixfct : callable, signature=(t, \*\*kwargs)
+    driftmatrixfct : callable, signature=(t, **kwargs)
```

Comment:
These changes cause warnings in Sphinx (see https://travis-ci.org/github/probabilistic-numerics/probnum/jobs/718931687):
```bash
/home/travis/build/probabilistic-numerics/probnum/src/probnum/filtsmooth/__init__.py:docstring of probnum.filtsmooth.LinearSDEModel:9: WARNING: Inline strong start-string without end-string.

/home/travis/build/probabilistic-numerics/probnum/src/probnum/filtsmooth/__init__.py:docstring of probnum.filtsmooth.LinearSDEModel:13: WARNING: Inline strong start-string without end-string.

/home/travis/build/probabilistic-numerics/probnum/src/probnum/filtsmooth/__init__.py:docstring of probnum.filtsmooth.LinearSDEModel:17: WARNING: Inline strong start-string without end-string.

/home/travis/build/probabilistic-numerics/probnum/src/probnum/filtsmooth/gaussfiltsmooth/unscentedtransform.py:docstring of probnum.filtsmooth.UnscentedTransform.propagate:8: WARNING: Inline strong start-string without end-string.
```

 and likely do not produce the docs output we are looking for.

Reply:
My bad, thought I had checked those things but apparently not for all of those string changes! I'll look into fixing that such that the builds look as they should, while pylint does not complain.

Reply:
Solved this in nathanaelbosch:probnum/ pylint @6e847a5